README.md

```diff
@@ -30,15 +30,17 @@ $ rake db:migrate
 $ rake server
 ```
 
-### Loading sample data
+This will start Solr, index some sample data, and run the application.
+
```

Comment:
I think the command got eaten?

Reply:
I meant that line to explain what the previous line (`rake server`) will do — that it does solr+data+rails.  Then there's commands below to show how to drop & reload the sample data.

Maybe this line should be above the `rake server` command to make that clearer?

Reply:
Ah! no it reads totally fine in full context!